### ISSUE TITLE
Homework 2

### ISSUE THREAD

**Issue by nachoaguadoc** [hediD/ada-assignments#2]
Hi! Pretty good job overall, although you haven't included any chart to back up your results and conclusions. As a note for the next homework peer review, I had my difficulties finding the "Data from the web.ipynb" file because it seems that you removed it a couple of days ago. I didn't have time to review it until today, so I had to browse through your commit history to find it. It isn't a problem at all for me, but maybe in future homeworks other reviewers do mind it and doesn't take the time to look into your commit history.

Cheers!

**Comment by hediD** [comment]
Yes I deleted it by mistake, really sorry ! Thank you for remaining me !
